titan-web-client/src/layouts/dashboard/components/sidebar/SidebarContentGroup.js

```diff
@@ -17,7 +17,7 @@ class SidebarContentGroup extends React.Component {
 }
 
 SidebarContentGroup.propTypes = {
-  children: PropTypes.object
+  children: PropTypes.any

```

Comment:
I don't believe `children` needs to be included in propTypes anymore. There should already be a lint rule to allow its usage without specifying it.